checklists/alz_checklist.en.json

```
@@ -2615,7 +2626,8 @@
             "guid": "cc87a3bc-c572-4ad2-92ed-8cabab66160f",
             "id": "H04.01",
             "severity": "High",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/considerations/landing-zone-security#secure"
+            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/considerations/landing-zone-security#secure",
+            "training": "https://learn.microsoft.com/training/paths/implement-security-through-pipeline-using-devops/"
```

@alisakina99 - while this training path is a valuable topic on the security of the pipelines & repos themselves, the associated recommendation from CAF is about improving cloud security posture by including the security team in the DevOps processes. That said, I would suggest another path that discusses automated code scanning and continuous security validation being included in the pipelines (see suggested link)

```suggestion
            "training": "https://learn.microsoft.com/training/paths/az-400-implement-security-validate-code-bases-compliance/"
```